### PR TITLE
Editor: Fixed sidebar glitches

### PR DESCRIPTION
**Pull request by ycw**
The issues: 

1. Editor's sidebar not wide enough for language `ja` , so on **first-run**, the property tabs will be wrapped:

image

2. Currently sidebar's width is unrecoverable, so say Japanese users have to resize the sidebar **every time** the editor restarted (including **refresh the page** for purging texture input's cache :D)

3. Resizer can be dragged way too far to the right, so for languages `fr` and `ja` the property tabs will be wrapped:

https://github.com/mrdoob/three.js/assets/1063018/4061c019-5e5b-4cb2-93d4-fb0590a6f9cc

4. Resizer can be dragged way too far to the left, so once users release the pointer, the resizer can't be dragged again because it's now out of window:

https://github.com/mrdoob/three.js/assets/1063018/d65a0869-e997-4d6b-80b9-cdfdd69ce330

This PR:

- Fixed 1 by detecting effective language on first-run, and then set proper width 
- Fixed 2 by (re)storing sidebar's width to Config
- Fixed 3 by restricting sidebar's min-width based on editor Settings>Language
- Fixed 4 by setting lower bound of sidebar's width to 10(px) instead of 0(px)
 
Preview: (open in incognito mode to mimic first-run) https://raw.githack.com/ycw/three.js/editor-sidebar-width-is-user-pref/editor/index.html

### CODE REVIEW

File: editor/js/Config.js [hunk 1]
```diff
@@ -3,10 +3,25 @@ function Config() {
...
 	const suggestedLanguage = [ 'fr', 'ja', 'zh' ].includes( userLanguage ) ? userLanguage : 'en';
 
+	function getSidebarMinWidth( language = suggestedLanguage ) {
+
+		return { // >= #sidebar min-width (335px)
+
+			'en': 350,
+			'fr': 355,
+			'ja': 395,
+			'zh': 335
```
Comment (mrdoob) [inline editor/js/Config.js:15]: This is yet more code that needs to be maintained.

I'm sure there's a solution for this issue that doesn't create more maintenance work.

Comment (ycw) [reply editor/js/Config.js:15]: css.

Comment (ycw) [reply editor/js/Config.js]: ok, i found a way to eliminate those numbers, by auto sniffing sum of tabs `offsetWidth` of the widest tabbed panel

**Comment by ycw** [comment]
done :D

**Comment by mrdoob** [comment]
Making the tabs horizontally scrollable may be a better solution.

**Comment by ycw** [comment]
> Making the tabs horizontally scrollable may be a better solution.

ok, in this way just few lines of changes in CSS

**Comment by ycw** [comment]
done. see #28467

**Pull request closed without merging**